xcdat/dataset.py

```diff
@@ -44,8 +44,9 @@ def open_dataset(
         and lower bounds. Otherwise, use the provided time coordinates, by
         default False.
     add_bounds: bool, optional
-        If True, add bounds for supported axes (T, X, Y) if they are missing in
-        the Dataset, by default False.
+        If True, add bounds for supported axes (X, Y, T) if they are missing in
+        the Dataset, by default True. Bounds are required for many xCDAT
+        features.
```

Comment:
I added that "Bounds are required for many xCDAT features." to explain why `add_bounds` defaults to `True`.